sound/hybrid_font.h

```diff
@@ -553,6 +559,11 @@ class HybridFont : public SaberBase, public Looper {
 	return;
       case EFFECT_PREON: SB_Preon(location); return;
       case EFFECT_POSTOFF: SB_Postoff(); return;
+      case EFFECT_ACCENT_SWING: Play(&SFX_swing, &SFX_swng); return;
```

Comment:
I think this (and also the next one) should set swing_player_
(At least that's what the code used to do.)

Reply:
Sorry, I don't follow.
EVENT_SWING is generated, and all the code for a swing runs, including swing_player_ processes, no?
Not sure what explicitly would go here that isn't handled already elsewhere.

Reply:
Unless I'm missing something, swing_player_ isn't set anywhere anymore.
There is code on line 303 and elsewhere which prevents overlap and changes the volume of the swing sound, but none of that works unless swing_player_ is set correctly.

Reply:
> none of that works unless swing_player_ is set correctly.

Ok, I think the Mono sound was what was confusing me, as that doesn't need overlap prevention? So I think the code just needs to specifically handle each sound (mono vs poly) in each effect's case, so that the poly version assigns swing_player_.
Changes made. Let me know if this is accomplishing things correctly?